gcp/copy_folder.py

```
@@ -234,17 +240,88 @@ def _dest_has_folder(svc, dest_folder_id, folder_name):
     return files[0]["id"] if files else None
 
 
-def _copy_file_with_backoff(svc, file, dest_folder_id, max_retries, max_backoff):
+def _copy_permissions(svc, src_id, dest_id):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_copy_permissions`

Add return type annotation: `None`

(ANN202)